docker/development/Dockerfile

```diff
@@ -0,0 +1,59 @@
+# Ubuntu 18.04 with nvidia-docker2 beta opengl support
+FROM nvidia/opengl:1.0-glvnd-devel-ubuntu18.04
+
+# install third-party packages for things like developemnt
+RUN apt-get -qq update && apt-get -y install \
+    git \
+    gnupg \
+    g++-8 \
+    lsb-release \
+    sudo \
+    vim \
+    wget \
+  && rm -rf /var/lib/apt/lists/*
+
+# set up repositories
+RUN sh -c 'echo "deb http://packages.osrfoundation.org/gazebo/ubuntu-stable `lsb_release -cs` main" > /etc/apt/sources.list.d/gazebo-stable.list' \
+  && sh -c 'echo "deb http://packages.osrfoundation.org/gazebo/ubuntu-prerelease `lsb_release -cs` main" > /etc/apt/sources.list.d/gazebo-prerelease.list' \
```

Comment:
Since this container is meant for building from source, I'd remove the `prerelease` repo